### DISCUSSION TITLE
Label really small objects

### DISCUSSION THREAD

**Discussion by thangphung215** [ultralytics/yolov5#3019] [Q&A]
Hi everybody, I am doing a project on detecting vehicle from the front view of my car but when I label object, there are so many tiny objects that I do not know whether necessary to label those tiny objects or only label some not so small objects. Thank you very much.

**Comment by glenn-jocher** [comment]
@thangphung215 YOLOv5 will learn to predict the same types and sizes of objects you have labelled. If you neglect to label small objects none will be detected during deployment.

**Reply by thangphung215** [reply]
> @thangphung215 YOLOv5 will learn to predict the same types and sizes of objects you have labelled. If you neglect to label small objects none will be detected during deployment.

Thank you very much
